julia.html.markdown

```diff
@@ -78,7 +78,7 @@ false
 1 < 2 < 3 # => true
 2 < 3 < 2 # => false
 
-# Strings are created with "
+# AbstractStrings are created with "
```

Comment:
I'm a bit -1 on this **line** itself being changed, because it's so confusing. If you want to keep this one change, care to add a "(Julia's basic string type" note or something?